### ISSUE TITLE
Marketplace

### ISSUE THREAD

**Issue by nakosung** [ncsoft/Unreal.js#183]
Marketplace should be updated. :)

**Comment by Youdaman** [comment]
Yes, the plugin on the marketplace is only compatible with UE4 4.15 when 4.17 is the latest version.

What is the delay with updating the marketplace with the latest version? Is it on the Epic side of things?

**Comment by crocuis** [comment]
Unreal.js 4.16 is registered in the Marketplace. Version 4.17 has a problem with the Mac version and may take some time to update.

**Comment by Youdaman** [comment]
It's a shame they don't let you release different versions for each platform. Is that possible?

**Comment by crocuis** [comment]
I fixed the problem and asked to be updated in the marketplace.

**Comment by Youdaman** [comment]
Wonderful news! Thank you @crocuis 👍

**Comment by crocuis** [comment]
Unreal.js 4.17 is registered in the Marketplace.  :)

**Comment by Youdaman** [comment]
@crocuis 🥇 Thank you sir!
